external/radiation/radiation/wrapper_api.py

```diff
@@ -162,14 +175,7 @@ def _rad_compute(self, state: State, time: cftime.DatetimeJulian,) -> Diagnostic
         grid, coords = get_grid(state)
         sfcprop = get_sfcprop(state)
         ncolumns, nz = statein["tgrs"].shape[0], statein["tgrs"].shape[1]
-        model = get_model(
-            self._rad_config,
-            self._tracer_inds,
-            time,
-            self._timestep,
-            nz,
-            self._comm.rank,
-        )
+        model = self._get_model(time, nz,)
```

Comment:
would `_get_model_config` be accurate? "Model" is already very overloaded for us...

Reply:
See comment below.